### PR TITLE
Update Content.js

### PR DESCRIPTION
**Pull request by stefantsonev2019**

### CODE REVIEW

File: Content.js [hunk 1]
```diff
@@ -17,16 +15,16 @@ export default function Postman({ handler, value, className, onCancel, onSave, r
   const [ submitted, submit ] = useState(false);
   const [ deleteSure, areYouSure ] = useState(false);
   const isEditing = !!value;
+  const something = 42;
```
Comment (krasimir) [inline Content.js:18]: You sure? !!

File: Content.js [hunk 2]
```diff
@@ -17,16 +15,16 @@ export default function Postman({ handler, value, className, onCancel, onSave, r
...
 
   const reset = () => {
-    submit(false);
+    submit(true);
```
Comment (krasimir) [inline Content.js]: I'm not sure about that. Maybe should be still `false` ❔

File: Header.js [hunk 3]
```diff
@@ -2,6 +2,9 @@ import React from 'react';
 
 export default function Header({ children }) {
   return (
-    <h3>This is a title here</h3>
+    <React.Fragment>
+      <h3>This is a title here</h3>
+      <p>Hello</p>
```
Comment (krasimir) [inline Header.js:7]: Can we come with better copy here.

**Review by krasimir — CHANGES_REQUESTED** [review]

File: Content.js (hunk 1 above)
Comment (stefantsonev2019) [reply Content.js:18]: Not really.

File: Header.js (hunk 3 above)
Comment (stefantsonev2019) [reply Header.js:7]: Ok, I'll think about it.

File: Content.js [hunk 4]
```diff
@@ -35,14 +33,14 @@ export default function Postman({ handler, value, className, onCancel, onSave, r
   return (
     <div className={ `postman cf ${ className }` }>
       <div className='media small'>
-        <img src={ profile.avatar } className='avatar bold' title={ profile.login }/>
```
Comment (krasimir) [inline Content.js:38]: Why is this removed?

File: Content.js (hunk 1 above)
Comment (stefantsonev2019) [reply Content.js:18]: That's just testing.

File: Content.js [hunk 5]
```diff
@@ -1,6 +1,4 @@
-/* A
-Comment here.
-*/
+/* A Comment here. */
```
Comment (krasimir) [inline Content.js:1]: Shall we go with:
```
// A Comment here 1!
```

File: Content.js [hunk 6]
```diff
@@ -35,14 +33,14 @@ export default function Postman({ handler, value, className, onCancel, onSave, r
...
         <textarea
           value={ text ? text : '' }
           placeholder='Reply'
           className={ text !== null ? 'type' : '' }
           onClick={ () => type(text || '') }
           disabled={ submitted }
           onChange={ e => type(e.target.value) } />
+        <img src={ profile.avatar } className='avatar bold' title={ profile.login }/>
       </div>
       { (isEditing && !submitted) && <div className='left mt05 ml2'>
```
Comment (krasimir) [inline Content.js]: One More comment.

File: Content.js (hunk 5 above)
Comment (krasimir) [reply Content.js:1]: Riew works

File: Content.js [hunk 7]
```diff
@@ -17,16 +15,16 @@ export default function Postman({ handler, value, className, onCancel, onSave, r
   const [ submitted, submit ] = useState(false);
   const [ deleteSure, areYouSure ] = useState(false);
```
Comment (krasimir) [inline Content.js:16]: A new comment but I can't see the delete and edit button.

File: Content.js (hunk 5 above)
Comment (krasimir) [reply Content.js:1]: nnn

**Pull request merged**